content/post/Android_Root101.en.md

```diff
@@ -85,13 +88,12 @@ _After completing the process make sure to clear the Dalvik Cache and Cache memo
 
 _Reboot the Device._
 
-**_Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares._**
+{{<yellowborder>}}Congratulations! Now you have a phone with a CUSTOM ROM of your own choice with no bloatwares.{{</yellowborder>}}
 
-______
 ______
 
-*If during the process you encoutered any issues do look up your issues on XDA or else Google your issues, you'll surely find a solution.*
+*If during the process you encoutered any issues, do look up your issues on XDA or else Google your issues, you'll surely find a solution.*
 
-*If you enjoyed reading this article do let me know by tweeting to me on my __twitter-handle @RohanVTK__. Your input will highly appreciable as well.*
+*Want to connect?* [📱Connect Here](https://twitter.com/RohanVTK).
```

Comment:
What I actually meant :laughing: 
```suggestion
*Want to connect?* [📱 Hit me up on Twitter @RohanVTK](https://twitter.com/RohanVTK)
```
1. Give them a reason to click the link
2. Give them an idea of what they are clicking without looking at the bottom left of their screen.